src/core/lib/iomgr/udp_server.cc

```diff
@@ -481,10 +483,10 @@ void GrpcUdpListener::OnRead(grpc_error* error, void* do_read_arg) {
   if (udp_handler_->Read()) {
     /* There maybe more packets to read. Schedule read_more_cb_ closure to run
      * after finishing this event loop. */
-    GRPC_CLOSURE_INIT(
-        &do_read_closure_, do_read, do_read_arg,
-        grpc_core::Executor::Scheduler(grpc_core::ExecutorJobType::LONG));
-    GRPC_CLOSURE_SCHED(&do_read_closure_, GRPC_ERROR_NONE);
+    GRPC_CLOSURE_INIT(&do_read_closure_, do_read, do_read_arg, nullptr);
```

Comment:
do we still need the last argument to `GRPC_CLOSURE_INIT`? It's always set to nullptr now, or are there any left overs?

Reply:
After this, all the remaining ones will grpc_schedule_on_exec_ctx :)

Reply:
Ah great!